dialplan/asterisk/extensions_lib_did.conf

```diff
@@ -20,7 +20,9 @@ same  =   n,AGI(agi://${XIVO_AGID_IP}/check_schedule)
 same  =   n,GotoIf($["${XIVO_SCHEDULE_STATUS}" = "closed"]?CLOSED,1)
 same  =   n,GoSub(xivo-subroutine,s,1(${XIVO_DIDPREPROCESS_SUBROUTINE}))
 same  =   n,Gosub(xivo-global-subroutine,s,1)
-same  =   n,Gosub(forward,s,1(${XIVO_FWD_INCALL_ANSWER_ACTION},${XIVO_FWD_INCALL_ANSWER_ACTIONARG1},${XIVO_FWD_INCALL_ANSWER_ACTIONARG2}))
+same  =   n,GotoIf(${WAZO_GREETING_SOUND}?:nogreeting)
+same  =   n,Playback(${WAZO_GREETING_SOUND})
```

Comment:
Be careful, playback is not async, so if playback file is 5 minutes, nothing happens when the file is playing.

Reply:
Yes it was the specification
> As administrator, on incoming call I want to configure a message completely before distributing to destination